### ISSUE TITLE
Wrong items selected for existing selections

### ISSUE THREAD

**Issue by waldermort** [silviomoreto/bootstrap-select#942]
I have two forms on the page, both contain an almost identical select control. The options are the same, but the selected attribute is added on the server.

```html
<select multiple="multiple" class="selectpicker form-control" name="data[5][roles][]" style="display: none;">
  <option value="2">ROLE_USER</option>
  <option selected="selected" value="3">ROLE_SUPER_ADMIN</option>
</select>

<select multiple="multiple" class="selectpicker form-control" name="data[6][roles][]" style="display: none;">
  <option selected="selected" value="2">ROLE_USER</option>
  <option value="3">ROLE_SUPER_ADMIN</option>
</select>
```

When the page is rendered in the browser, both controls have the ROLE_SUPER_ADMIN option selected. As you can imagine, submitting the form and persisting to the database is going to leave me with a bunch of super admins I don't want :/

**Comment by caseyjhol** [comment]
And this works fine without bootstrap-select? If you unhide the select, is the correct value actually selected? What is the text in the button displaying? Some more details would be great.

**Comment by waldermort** [comment]
@caseyjhol Yes, it works fine without bootstrap-select. Looking at the source, I see the original select controls are hidden, but still have the server set attributes. It is the bootstrap-select generated control which is displaying the wrong selections. I was wrong above when I stated that submitting the form would have the wrong results, it is only the web browser display which is wrong.

I tried bootstrap-select as a quick fix last night, but when I noticed it wasn't working as supposed to I had to put it asside, so I didn't do much in the line of debugging.

**Comment by waldermort** [comment]
@caseyjhol Sorry again. But after retrying, I noticed the problem persisted. This time though, I have found the cause to be the 'autocomplete' attribute. When autocomplete is not configured (defaults to on) select-picker will select the values that the browser has inserted rather than those which have been configured on the server. A workaround is to add the attribute `autocomplete='off'` to the select control. I'm not sure if this is really the libraries problem or not. Either way, I think a note should be added to the documents about it.
